port/ztpf/omrosbacktrace_impl.c

```diff
@@ -149,6 +151,10 @@ omrintrospect_backtrace_thread_raw(struct OMRPortLibrary *portLibrary,
 	ret = backtrace_sigprotect(portLibrary, threadInfo, addresses,
 		sizeof(addresses) / sizeof(void*));
 
+	//z/TPF Stack grows down.  The top-of-stack has a dummy frame which should be ignored.
+	//Therefore reducing stack frame count by one.
```

Comment:
`//` is not a proper C comment. Can you switch the comment to `/* */` style.

Reply:
Done.